guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<ol><li>For those bold text, shall we keep the lowercase/uppercase consistent? Right now, it is "&nbsp;<strong>Object Detection" and "Object instance segmentation".</strong></li><li><strong>Can we add hyperlink for "how unet works" as well?</strong></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>